server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CICSParser.g4

```diff
@@ -927,11 +979,11 @@ cics_value: LPARENCHAR ptr_value RPARENCHAR;
 empty_parens: LPARENCHAR RPARENCHAR;
 
 cobolWord
-   : IDENTIFIER | cics_only_words | db2sql_only_words
+   : IDENTIFIER | cics_only_words | db2sql_only_words | cobolCompilerDirectivesKeywords
```

Comment:
In future, we should clean-up / separate these parsers as well (e.g. CICSParser misleading with non-related cics rules)